po/it.po

```diff
@@ -88,60 +70,66 @@ msgstr ""
 "Selezionare un'applicazione per aprire «%s». Ulteriori applicazioni sono "
 "disponibile in «<a href='software'>Software</a>»."
 
-#: src/appchooserdialog.c:378
+#: ../src/appchooserdialog.c:378
 msgid ""
 "Select an application. More applications are available in <a "
 "href='software'>Software.</a>"
 msgstr ""
-"Selezionare un'applicazione. Ulteriori applicazioni sono disponibile in "
-"«<a href='software'>Software</a>»."
+"Selezionare un'applicazione. Ulteriori applicazioni sono disponibile in «<a "
+"href='software'>Software</a>»."
 
-#: src/appchooserdialog.c:391
+#: ../src/appchooserdialog.c:391
 #, c-format
 msgid "Unable to find an application that is able to open “%s”."
 msgstr "Impossibile trovare una applicazione che possa aprire «%s»."
 
-#: src/appchooserdialog.c:396
+#: ../src/appchooserdialog.c:396
 msgid "Unable to find a suitable application."
 msgstr "Impossibile trovare una applicazione adatta."
 
-#: src/appchooserdialog.ui:13
-msgid "Open With"
-msgstr "Apri con"
-
-#: src/appchooserdialog.ui:166
-msgid "No Applications Found"
-msgstr "Nessuna applicazione trovata"
-
-#: src/appchooserdialog.ui:182
-msgid "Find Compatible Applications in Software"
-msgstr "Trova applicazione compatibile in «Software»"
-
-#: src/filechooser.c:363
+#: ../src/filechooser.c:363
 msgid "_Open"
 msgstr "A_pri"
 
-#: src/screenshotdialog.c:93
+#: ../src/filechooser.c:365
+msgid "_Cancel"
+msgstr "A_nnulla"
+
+#: ../src/screenshotdialog.c:93
 #, c-format
 msgid "Share this screenshot with %s?"
 msgstr "Condividere questa schermata con %s?"
 
-#: src/screenshotdialog.c:96
+#: ../src/screenshotdialog.c:96
 msgid "Share this screenshot with the requesting application?"
 msgstr "Condividere questa schermata con l'applicazione che la richiede?"
 
-#: src/screenshotdialog.ui:10
-msgid "Screenshot"
-msgstr "Schermata"
+#~ msgid "Share Details"
```

Comment:
This doesn't seem right: you're **removing** this translation, not updating it. Same for all the other strings below (`Share`, "Make changes before..."...).

Reply:
Hmmm, sorry... I didn't notice that.
What is the correct Make command to extract the translations and update the base file?

Reply:
Sorry, I did miss this message. I think you're looking for `make -C po/ update-po`